cppcache/integration-test/ThinClientHelper.hpp

```diff
@@ -134,7 +134,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3",
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4", "New Value-5", "New Value-6"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```

Comment:
One more instance

Reply:
Lots of tests use this header. Changing to a scalar caused lots of compile issues (mostly in disabled tests). Suggest we leave this file untouched, as probably too much work to fix compile issues.